### ISSUE TITLE
subtle v0.3 error

### ISSUE THREAD

**Issue by croaky** [miscreant/miscreant.rs#9]
Howdy @tarcieri @hdevalence,

I started seeing these errors within the last 24 hours:

```
error: no matching version `^0.3` found for package `subtle`
location searched: registry `https://github.com/rust-lang/crates.io-index`
versions found: 2.0.0, 1.0.0
required by package `miscreant v0.3.0`
```

It looks like subtle packages < v1 have been yanked?
https://crates.io/crates/subtle/versions

And the released versions of miscreant depend on the yanked series?
https://crates.io/crates/miscreant

Do you have guidance on how to resolve? Use GitHub as the Miscreant source in the interim?

Thanks!

**Comment by hdevalence** [comment]
This is my fault, I yanked the prerelease versions of subtle. I thought that I'd already sent a PR to Miscreant to fix this, but I've just unyanked the 0.3.0 version as a fix for now

**Comment by tarcieri** [comment]
Miscreant is updated, I just haven't done a release. I will try to do that Really Soon Now.

**Comment by hdevalence** [comment]
It looks like the git version is using `subtle` v2 as you mention (https://github.com/miscreant/miscreant.rs/blob/master/Cargo.toml#L31) but it's just not released, I should have checked that first!

**Comment by croaky** [comment]
No worries! Thanks for the quick response.

FWIW, it looks like https://github.com/dalek-cryptography/x25519-dalek/pull/20 v0.7 for `subtle` is a similar issue.

Will leave this issue for you to close as desired @tarcieri

**Comment by tarcieri** [comment]
Closed by #12
